tools/UpdatePaths/Scripts/76335_remove_caseless.txt

```diff
@@ -0,0 +1,3 @@
+#replaced by elements
+
+/obj/projectile/caseless/@SUBTYPES : /obj/projectile/@SUBTYPES{@OLD}
```

Comment:
```suggestion
/obj/item/ammo_casing/caseless/@SUBTYPES : /obj/item/ammo_casing/@SUBTYPES{@OLD}
```

this is my bad, i fed you an example of what i wanted in relation to the path changes you were making, not the actual changes.

Reply:
Ahaha, I hadn't noticed that, was too busy sipping energy drinks.